controllers/nugus_controller/nugus_controller.cpp

```diff
@@ -67,24 +67,26 @@ class NUgus : public webots::Robot {
             }
             else if (num_ready > 0) {
                 // Wire format
-                // unit64_t N   message size in bytes
+                // unit32_t N   message size in bytes in network byte order
```

Comment:
is this what you're trying to convey?
```suggestion
                // unit32_t Nn  message size in bytes. The bytes are in network byte order (big endian)
```